config/email-config-schema.yaml

```diff
@@ -3,7 +3,7 @@ required: ["bridge"]
 properties:
     bridge:
         type: "object"
-        required: ["domain", "homeserverUrl", "mxDomain", "mailPort"]
+        required: ["domain", "homeserverUrl", "mxDomain", "mailPort", "dkimEnabled", "startTLS"]
```

Comment:
I believe there is a way to do dependencies, so you must specify TLSKey / TLSCert if you enable startTLS (https://github.com/json-schema/json-schema/issues/158#issuecomment-86124035). It took me ages to Google for mind.

Another more straight-forward option is to just have a `tls` object which contains tlskey and tlscert properties which are both requried. If the `tls` object is not specified, tls is disabled